### PR TITLE
refactor: ui changes

### LINKED ISSUE #991: Transactions History (UI)
- [x] Recent Transactions font size (should be same as Overview title)
- [x] Send and Delegate Button of Same Length
- [x] Message tile should have more padding and square
- [x] Remove View all
- [x] space between double arrow icon and vertical line
- [x] Increase the space between total balance and amount
- [x] Format the amount
- [x] Available to Available Balance
- [x] Transaction item vertical divider opacity

### PR DESCRIPTION
**Pull request by Teja2045**
closes #991 

## Summary by CodeRabbit

- **New Features**
  - Introduced a new `formatNumber` function for better number formatting across transactions.

- **Style**
  - Updated various text and button styles for consistency and readability.
  - Added a new style for disabled primary buttons.

- **Refactor**
  - Improved code readability and maintainability in utility functions.

- **Documentation**
  - Updated component documentation to reflect text and link removals.

- **Bug Fixes**
  - Adjusted vertical spacing in transaction items to fix alignment issues.

- **Chores**
  - Cleaned up unused styles and code comments for a cleaner codebase.

- **Revert**
  - Removed "View All" links and certain button functionalities to revert to previous design decisions.

### CODE REVIEW

File: frontend/src/app/(routes)/(overview)/overview-components/History.tsx [hunk 1]
```diff
@@ -57,15 +54,15 @@ const Balance = ({ chainIDs }: { chainIDs: string[] }) => {
...
+          className="primary-custom-btn w-[144px]"
           onClick={() => {
             router.push(getPath(chainIDs, 'transfers'));
           }}
         >
           Send
         </button>
         <button
-          className="primary-custom-btn"
+          className="primary-custom-btn w-[144px]"
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview-components/History.tsx:65]: Don't use fixed length for buttons.

File: frontend/src/app/(routes)/(overview)/overview-components/History.tsx [hunk 2]
```diff
@@ -57,15 +54,15 @@ const Balance = ({ chainIDs }: { chainIDs: string[] }) => {
       </div>
       <div className="flex justify-center gap-6">
         <button
-          className="primary-custom-btn"
+          className="primary-custom-btn w-[144px]"
           onClick={() => {
             router.push(getPath(chainIDs, 'transfers'));
           }}
         >
           Send
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview-components/History.tsx:62]:
```suggestion
          &nbsp;&nbsp;Send&nbsp;&nbsp;
```

File: frontend/src/app/(routes)/(overview)/overview-components/History.tsx [hunk 3]
```diff
@@ -57,15 +54,15 @@ const Balance = ({ chainIDs }: { chainIDs: string[] }) => {
       </div>
       <div className="flex justify-center gap-6">
         <button
-          className="primary-custom-btn"
+          className="primary-custom-btn w-[144px]"
```
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview-components/History.tsx:57]:
```suggestion
          className="primary-custom-btn"
```

File: frontend/src/app/(routes)/(overview)/overview-components/History.tsx (hunk 1 above)
Comment (aleem1314) [inline frontend/src/app/(routes)/(overview)/overview-components/History.tsx:65]:
```suggestion
          className="primary-custom-btn"
```

**Review by charymalloju — APPROVED** [review]

**Pull request merged**